packages/now-cli/src/util/dev/router.ts

```diff
@@ -40,28 +41,21 @@ export default async function(
     let idx = -1;
     for (const routeConfig of routes) {
       idx++;
-      let { src, headers, methods, handle } = routeConfig;
-      if (handle) {
-        if (handle === 'filesystem' && devServer) {
+      if (isHandler(routeConfig)) {
+        if (routeConfig.handle === 'filesystem' && devServer) {
           if (await devServer.hasFilesystem(reqPathname)) {
             break;
           }
         }
         continue;
       }
 
+      let { src, headers, methods } = routeConfig;
+
       if (Array.isArray(methods) && reqMethod && !methods.includes(reqMethod)) {
         continue;
       }
 
-      if (!src.startsWith('^')) {
```

Comment:
Why are these removed?

Reply:
Because its handled in `@now/routing-utils`

Reply:
Take a look here: https://github.com/zeit/now/blob/110e1445fa267105fea86554f1d4b2825ff07791/packages/now-routing-utils/src/index.ts#L59